presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/ScalarAggregationToJoinRewriter.java

```diff
@@ -43,6 +43,7 @@
 import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Iterables;
```

Comment:
I'd change the title of this commit to "Preserve output shape in ScalarAggregationToJoinRewriter"